drivers/SmartThings/zwave-switch/profiles/zooz-zen-30-dimmer-relay.yml

```diff
@@ -0,0 +1,280 @@
+name: zooz-zen-30-dimmer-relay
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: powerMeter
+    version: 1
+  - id: energyMeter
+    version: 1
+  - id: colorControl
+    version: 1
```

Comment:
Those lines should be removed since device does not support colorControl capability

Reply:
> Those lines should be removed since device does not support colorControl capability

indeed, done